src/diceBot/Kemono.rb

```diff
@@ -0,0 +1,346 @@
+          【大木】耐久度7,軽減値3,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-2の修正を付加
+          【岩】耐久度6,軽減値4,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1の修正を付加/コンタクト内で行われる格闘攻撃のダメージ+1
+          【岩】耐久度6,軽減値4,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1の修正を付加/コンタクト内で行われる格闘攻撃のダメージ+1
+          【岩壁】耐久度8,軽減値4,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-2の修正を付加/コンタクト内で行われる格闘攻撃のダメージ+2
+          【岩壁】耐久度8,軽減値4,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-2の修正を付加/コンタクト内で行われる格闘攻撃のダメージ+2
+        }
+      },
+
+      'RIMT' => {
+        :name => 'ランダム遮蔽物表(屋内)',
+        :table => %w{
+          【木材の壁】耐久度4,軽減値2,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1の修正を付加
+          【木材の壁】耐久度4,軽減値2,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1の修正を付加
+          【木材の壁】耐久度4,軽減値2,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1の修正を付加
+          【木材の壁】耐久度4,軽減値2,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1の修正を付加
+          【木材の扉】耐久度4,軽減値2,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1、接触判定と突撃判定に-2の修正を付加
+          【木材の扉】耐久度4,軽減値2,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1、接触判定と突撃判定に-2の修正を付加
+          【木材の扉】耐久度4,軽減値2,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1、接触判定と突撃判定に-2の修正を付加
+          【木材の扉】耐久度4,軽減値2,特殊効果:コンタクト内のキャラクターに対する射撃攻撃判定に-1、接触判定と突撃判定に-2の修正を付加
+          【木製家具】耐久度3,軽減値2,特殊効果:コンタクト内で行われる格闘攻撃のダメージ+1
+          【木製家具】耐久度3,軽減値2,特殊効果:コンタクト内で行われる格闘攻撃のダメージ+1
+          【木製家具】耐久度3,軽減値2,特殊効果:コンタクト内で行われる格闘攻撃のダメージ+1
+          【木製家具】耐久度3,軽減値2,特殊効果:コンタクト内で行われる格闘攻撃のダメージ+1
+        }
+      },
+
+      'EET' => {
+        :name => '逃走体験表',
+        :table => %w{
+          【余裕】が0になる
+          【余裕】が0になる
+          【余裕】が0になる
+          任意の【絆】を合計2点減少する
+          任意の【絆】を合計2点減少する
+          任意の【絆】を合計2点減少する
+          全ての荷物を失う（逃走したエリアに配置され、調査で発見可能）
+          全ての荷物を失う（逃走したエリアに配置され、調査で発見可能）
+          全ての荷物を失う（逃走したエリアに配置され、調査で発見可能）
+          全ての武器と防具と小道具と荷物を失う（逃走したエリアに配置され、調査で発見可能）
+          全ての武器と防具と小道具と荷物を失う（逃走したエリアに配置され、調査で発見可能）
+          全ての武器と防具と小道具と荷物を失う（逃走したエリアに配置され、調査で発見可能）
+        }
+      },
+
+      'GFT' => {
+        :name => '食材採集表',
+        :table => %w{
+          食べられる根（栄養価:2）
+          食べられる根（栄養価:2）
+          食べられる草（栄養価:3）
+          食べられる草（栄養価:3）
+          食べられる草（栄養価:3）
+          食べられる実（栄養価:5）
+          食べられる実（栄養価:5）
+          食べられる実（栄養価:5）
+          小型動物（栄養価:10）
+          小型動物（栄養価:10）
+          大型動物（栄養価:40）
+          気持ち悪い虫（栄養価:1）
+        }
+      },
+
+      'GWT' => {
+        :name => '水採集表',
+        :table => %w{
+          汚水
+          汚水
+          汚水
+          汚水
+          汚水
+          汚水
+          飲料水
+          飲料水
+          飲料水
+          飲料水
+          飲料水
+          毒水
+        }
+      },
+
+      'WST' => {
+        :name => '白の魔石効果表',
+        :table => %w{
+          役に立たないものの色を変える
+          役に立たないものを大きくする
+          役に立たないものを小さくする
+          役に立たないものを保存する
+          役に立たないものを復元する
+          役に立たないものを召喚する
+          役に立たないものを動かす
+          役に立たないものを増やす
+          役に立たないものを貼り付ける
+          役に立たないものを作り出す
+          小型動物を召喚する
+          大型動物を召喚する
+        }
+      }
+    }
+
+  setPrefixes([/^K[AC]\d[-+\d]*/i, /CTR/i] + @@tables.keys)
```

Comment:
ここで `^` を使ってしまうとシークレットダイスに反応しないと思います。

Reply:
この修正と合わせて、シークレットダイスのテストケースの追加もお願いします。

Reply:
ご指摘のとおりですね。。修正します。

Reply:
合わせて、各項目を文字列で指定してください。

```suggestion
  setPrefixes(['K[AC]\d[-+\d]*', 'CTR'] + TABLES.keys)
```

Reply:
コマンドの修正、テストの追加を行いました。
（ `S` を頭につけたシークレットダイスのテストケースの追加、ついでに既存のケースに小文字をランダムに混ぜています）